### ISSUE TITLE
Architecture as Code: Glossary Definition

### ISSUE THREAD

**Issue by rocketstack-matt** [finos/architecture-as-code#254]
### Agree a Common Glossary of Terms
As discussed in the [Working Group Meeting on 8th August 2023](https://github.com/finos/devops-automation/issues/108), it was agreed that one of the first actionable items the working group should focus on is to agree a common glossary of terms that we will rely on. This will help ensure a common language and enable the group to effectively make prioritisation decisions.

This issue will track the initial creation of the glossary.

Once the initial glossary is complete (target mid-September, 2023) then this will form the foundation of the group voting on it's priorities and inform how many topics the group will work on and how we will organize to tackle them.

### How to Help
- [x] Create an initial set of terms to agree on, these are to be based on the 'What could (or should) we focus on?' section (page 5) of the [meeting materials](https://github.com/finos/devops-automation/files/12294129/2023-08-08.pdf) - @rocketstack-matt 
- Each definition has been created as an Issue to enable people who are not a member of the FINOS GitHub org to be able to participate. You can see all the definitions under discussion [here](https://github.com/finos/devops-automation/issues?q=is%3Aopen+label%3Aglossary+label%3Aarchitecture-as-code)
    - If you would like to propose a change of wording / clarification, please comment on the issue.
    - If you would agree with a definition, either the original or an amendment proposed in the comments the please vote on it using the 👍 emoticon.
- [ ] If there are additional terms you would like to contribute to the glossary, please create a new Issue using the [Glossary Definition](https://github.com/finos/devops-automation/issues/new?assignees=&labels=architecture-as-code%2C+glossary&projects=&template=Glossary.md) template.
